src/main/java/seedu/duke/command/AddCommand.java

```diff
@@ -21,17 +21,44 @@ public AddCommand(Ui ui, Data data, HashMap<String, String> arguments) {
 
     @Override
     public void execute() throws Exception {
-        // TODO Replace by ui after ui is implemented
+
         String patientID = arguments.get("payload");
+        patientID = patientID.toUpperCase();
         int stringLength = patientID.length();
+        boolean validID = true;
 
-        // TODO more ways to check for invalid NRIC and other exceptions
+        // Checks if ID has 9 characters
         if (stringLength != 9) {
```

Comment:
Would it be better to extract this as a global constant to avoid using magic values?

Reply:
I have extracted all the magic values and used global constants instead